src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -1271,10 +1271,22 @@ namespace
         NGRAPH_CHECK(lhs.getType().isa<MemRefType>());
         Type elemTy = lhs.getType().dyn_cast<MemRefType>().getElementType();
 
+        auto origOperand = op->getOperands()[0];
+        auto origIntType =
+            origOperand.getType().cast<NGTensorType>().getElementType().dyn_cast<IntegerType>();
+
         AffineLoopNestBuilder(ivs, lbs, ubs, steps)([&] {
             Value val = iLHS(ivs);
             Value zero = createZeroConstant(elemTy);
-            iRes(ivs) = std_select(val > zero, val, zero);
+            auto cmp = val > zero;
+            auto builder = ScopedContext::getBuilderRef();
```

Comment:
`rewriter` is a special builder that is available in this function. We can use it here instead of retrieving a new builder.

Reply:
Done.